.github/workflows/recurring-campaign-base.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: recurring-campaign-base
+
+permissions:
+  contents: read
+
+on:
+  workflow_call:
+    inputs:
+      campaign_identifier:
+        description: "The identifier of the campaign to run"
+        required: true
+        type: string
+    secrets:
+      github_token_for_moderne:
+        description: "The GitHub token to use when generating pull requests"
+        required: true
+      moderne_token:
+        description: "The Moderne token to use to authenticate with Moderne"
+        required: true
+      gpg_public_key:
+        description: "The GPG public key to use when signing commits"
+        required: true
+      gpg_private_key:
+        description: "The GPG private key to use when signing commits"
```

Comment:
Why must the private key and it's password be shared?

Reply:
Not entirely sure, but Moderne requires all three to be sent via their API. Otherwise it doesn't work. Feel free to ask in the Moderne slack channel